tests/e2e/cucumber/steps/api.ts

```diff
@@ -22,6 +22,7 @@ Given(
     const admin = this.usersEnvironment.getUser({ key: stepUser })
     for await (const info of stepTable.hashes()) {
       const user = this.usersEnvironment.getUser({ key: info.id })
+      await api.provision.unAssignRole({ admin, user })
```

Comment:
Please, add a comment here explaining why we need to unassign role and for which setup